### DISCUSSION TITLE
How to draw ApexChart in embed elements

### DISCUSSION THREAD

**Discussion by pbpb13** [Budibase/budibase#4808] [Help]
I want to draw ApexChart in Embed elements because the data have two series.
Example Data
Year      Income     Outcome
2020         212            124
2021         367            231
2022         568           432

How to manual draw ApexChart in Embed elements.

**Comment by mjashanks** [comment]
I'm not familiar with Apex charts. If it allows you to embed using an external script, it should work:

e.g. `<script src="/path/to/apex.js?parameters={{bindings}}">`

**Comment by pbpb13** [comment]
@mjashanks I found apexcharts script is contain in budibase-client.js but can't call ApexCharts function to draw chart.

**Reply by mjashanks** [reply]
Yep - we use it for our charts. Javascript in Budibase is sandboxed - you don't have access to the DOM. So this is probably why it's not possible.
